packages/eslint-plugin-boxel/lib/utils/import-utils.js

```diff
@@ -0,0 +1,163 @@
+/** @type {import('eslint').Rule.RuleModule} */
+
+/**
+ * Adds an import statement for a missing import, or augments an existing import statement
+ * @param {import('eslint').Rule.RuleFixer} fixer The fixer instance
+ * @param {import('eslint').SourceCode} sourceCode The source code object
+ * @param {string} consumedName The local name that will be used in the code
+ * @param {string} exportedName The name that is exported from the module
+ * @param {string} module The module to import from
+ * @returns {import('eslint').Rule.Fix | null}
+ */
+function fixMissingImport(
+  fixer,
+  sourceCode,
+  consumedName,
+  exportedName,
+  module,
+) {
+  // Check if an import from this module already exists
+  const importDeclarations = sourceCode.ast.body.filter(
+    (node) =>
+      node.type === 'ImportDeclaration' &&
+      node.source.value === module &&
+      // Skip type-only imports
+      node.importKind !== 'type',
+  );
+
+  if (importDeclarations.length > 0) {
+    // Module is already imported, so add to existing import
+    // Use the first non-type import declaration
+    const existingImport = importDeclarations[0];
+
+    // If it's a default import that we need to add
+    if (exportedName === 'default') {
+      // Check if the default import already exists
+      const hasDefaultImport = existingImport.specifiers.some(
+        (specifier) => specifier.type === 'ImportDefaultSpecifier',
+      );
```

Comment:
I'm a little confused about how you could fix a default export. For named exports we can see that the delcaration has a meaningful name like `CardDef`. so we can make some good guesses about what to import. In the case of a default export, it could be named anything. also in the case of a default export that already exists are you also confirming that the declaration for the default export is the same name that you are already using?

```ts
import Foo from 'foo'`;

foo.blah();

bar.blah(); // this is also a default export from foo that you discovered but its using a different declaration
```